Makefile

```diff
@@ -203,7 +203,7 @@ pwqfilter.o: pwqfilter.c passwdqc_filter.h passwdqc.h
 	$(CC) $(CPPFLAGS_lib) $(CFLAGS_lib) -c $*.c
 
 $(PKGCONFIG): $(PKGCONFIG).in
-	sed -e "s|@VERSION@|$(VERSION)|g" $< > $@
+	sed -e "s|@VERSION@|$(VERSION)|g" $(PKGCONFIG).in > $@
```

Comment:
Are you porting this to a make that doesn't support `$<` in explicit rules?

Reply:
Yes. I was getting empty string in place of `$<` here with Solaris' native `make`.